app/src/main/java/grinnell/appdev/edu/lyles/ItemAdapter.java

```diff
@@ -50,7 +50,7 @@ public void onClick(View view) {
                 this.getAdapter().expandContractItem((ViewHolder) mRecyclerView.findViewHolderForAdapterPosition(expandedIndex));
             }
             this.getAdapter().expandContractItem(this);
-            expandedIndex = this.getAdapterPosition();
+            expandedIndex = (expandedIndex == this.getAdapterPosition())? -1: this.getAdapterPosition();
```

Comment:
Please define fields before they are used. This is a slightly uncommon case because there's a subclass, but move the definition of `expandedIndex` and `mRecyclerView` to above the definition of `ViewHolder`.